src/components/breadcrumb/examples/Breadcrumb.vue

```diff
@@ -9,25 +9,33 @@
     <cdr-breadcrumb
       :items="averageBreadcrumbItems"
       data-backstop="breadcrumbs-ellipsis"
+      aria-label="two-item-breadcrumb"
+      id="two-item-breadcrumb"
```

Comment:
Two issues here:

1. If an `id` is not passed, one is auto-generated at runtime. Passing an id ensures it's not unique.
2. The parent element of a breadcrumb is a `nav`, when multiple breadcrumb components are on the page (not standard), the `nav` landmark is not uniquely labelled. Passing an aria-label solves that.